### PR TITLE
chore: github action for binary build

### PR DESCRIPTION
**Pull request by fanweixiao**
Run this script can generate binaries

```bash
PLATFORMS=darwin/amd64,darwin/arm64,windows/amd64,windows/arm64,linux/amd64,linux/arm64 bash build.sh
```

### CODE REVIEW

**Comment by venjiang** [comment]
- TLS expires
- When creating the Zipper, Downstreams and Auth Options are not added. The development mode has no problem, but need to configure this parameter in the production environment
- 'yomorun/psig' and `prscd` both references `yomo`, it is easy to cause `yomo` version conflict, recommended to put it directly in the `prscd` repository

**Comment by fanweixiao** [comment]
@venjiang Yes, you're right. I will merge `psig` to this repo after merging this PR.

**Review by venjiang — APPROVED** [review]

**Review by venjiang — COMMENTED** [review]
good

**Review by woorui — APPROVED** [review]

**Pull request merged**
